README.md

```diff
@@ -1,5 +1,77 @@
 # SQL Exercises
-
+# Core SQL
+
+## Challenge Rating
+
+This goal will likely be within your ZPD if you...
+
+- Are familiar with basic command line operations
+- Are interested in learning about writing SQL queries
+- Are interested in learning how to install postgres & use the `psql` command line tool.
+
+## Description
+
+A day-by-day structured deep dive into the basics of SQL. Complete lessons, and read tutorials.
+
+This goal makes heavy use of external resources: the [Introduction to SQL](https://www.khanacademy.org/computing/computer-programming/sql) course on [Khan Academy](https://www.khanacademy.com/), [Tutorial's Point](https://www.tutorialspoint.com/postgresql), [SQL Zoo](http://sqlzoo.net) and [SQL Bolt](https://sqlbolt.com/lesson/select_queries_order_of_execution)
+
+For the first few days, you'll be working on existing courses and completing tutorials. Then you'll use your skills to complete a set of SQL exercises and submit your queries as your artifact.
+
+Fork the [sql-exercises][sql-exercises] repo, and use the fork as your artifact. Complete all the exercises without looking at solutions on the internet.
+
+## Context
+
+In order to build web applications, it's important to get a solid grounding into the basics of SQL. Proficiency in SQL is a core skill needed to build the backend.
+
+Becoming a full stack web developer means that you need to be comfortable with writing SQL queries and understanding relational database design.
+
+## Specifications
+
+### Day 1: SQL Basics & Advanced SQL Queries
+- [X] Complete the [SQL Basics course](https://www.khanacademy.org/computing/computer-programming/sql#sql-basics)
+- [X] Complete the [Advanced SQL queries course](https://www.khanacademy.org/computing/computer-programming/sql#more-advanced-sql-queries)
+
+### Day 2: Relational Queries
+- [X] Complete the [Relational Queries course](https://www.khanacademy.org/computing/computer-programming/sql#relational-queries-in-sql)
+- [X] Complete the [Modifying databases with SQL course](https://www.khanacademy.org/computing/computer-programming/sql#modifying-databases-with-sql)
+
+### Day 3: Install Postgres and Complete Tutorials with psql
+- [X] Install Homebrew by following instructions listed [here](https://gist.github.com/punitrathore/ca32542fddd0d8b625aab610c35e4545)
+- [X] Install Postgres by following instructions listed [here](https://gist.github.com/punitrathore/ca32542fddd0d8b625aab610c35e4545#install-postgres)
+- [ ] Learn the `psql` command line tool by working through the following tutorials
```

Comment:
Did you not complete this task? It seems like you did, but you forgot to check off this box?